### ISSUE TITLE
scalePercentBy100 is not actually exposed as a public method

### ISSUE THREAD

**Issue by MickeyKay** [adamwdraper/Numeral-js#628]
Unless I'm mistaken, it's impossible to use `scalePercentBy100` as it is not publicly exposed like other methods here: https://github.com/adamwdraper/Numeral-js/blob/7de892ffb438af6e63b9c4f6aff0c9bc3932f09f/src/numeral.js#L452-L474

This should be a simple fix, want me to file a PR?

**Comment by MrCsabaToth** [comment]
True. But you can set it globally by executing `numeral.options.scalePercentBy100 = false;` if that suits your needs.

**Comment by pmairoldi** [comment]
I've already created a pull request for this see (#565).
